src/coreclr/src/vm/ceemain.cpp

```diff
@@ -1188,8 +1188,7 @@ void InnerCoEEShutDownCOM()
 
     g_fShutDownCOM = true;
 
-    // Release IJupiterGCMgr *
-    RCWWalker::OnEEShutdown();
+    Interop::OnRuntimeShutdown();
```

Comment:
Is the shutdown doing anything important - what is not going to now work if we delete this?

We strongly prefer to avoid doing work during shutdown.

Reply:
Minor cleanup. I can definitely avoid it without much issue. The main thing that is occurring is the release of the tracker runtime manager (i.e. `IReferenceTrackerManager`). This is the contract that the Jupiter runtime would provide us during our initial handshake.

Reply:
We do not suspect other threads during shutdown. There is a chance that other thread may be using this is running, unless there are locks etc. to prevent it.

Reply:
>There is a chance that other thread may be using this is running, unless there are locks etc. to prevent it.

Only managed threads and the GC thread use the set value. Can either of those types of threads be running when this function is called?

Reply:
What a disaster. The `InnerCoEEShutDownCOM()` is no longer called. @jkotas thoughts on removing the entire thing in this PR or in another?

Reply:
I have opted to remove it in a new PR. I will update this one as soon as the removal is merged in - see https://github.com/dotnet/runtime/pull/32381.